### PR TITLE
Update CLI plug-in docs around creating user profiles

### PR DESCRIPTION
**Pull request by zFernand0**
Describe your pull request here: Removed the V1 `zowe profiles` commands when creating profiles for the MQ and FTP plug-ins

List the file(s) included in this PR:  `docs/user-guide/cli-ftpplugin.md` and `docs/user-guide/cli-mqplugin.md`

After creating the PR, follow the instructions in the comments.

### CODE REVIEW

File: docs/user-guide/cli-ftpplugin.md [hunk 1]
```diff
@@ -39,45 +39,26 @@ Use one of the following methods to install or update the plug-in:
 
 ## Creating a user profile
 
-You create a zftp profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
```
Comment (anaxceron) [inline docs/user-guide/cli-ftpplugin.md:42]: Line 3, Line12: data sets should be two words
Line 16: Can you add "note" immediately after the three colons? (As is: **:::note**) I think I messed up the syntax here for the admonition. My bad.

File: docs/user-guide/cli-ftpplugin.md [hunk 2]
```diff
@@ -39,45 +39,26 @@ Use one of the following methods to install or update the plug-in:
...
+After you install the plug-in, create an FTP profile. An FTP profile is recommended to issue commands to via FTP. FTP profiles contain your host, port, user name, and password for connect to z/OS using FTP. You can create multiple profiles and switch between them as needed.
```
Comment (anaxceron) [inline docs/user-guide/cli-ftpplugin.md:42]: There're some extra words/odd tenses here: "An FTP profile is recommended to issue commands **to via** FTP. FTP profiles contain your host, port, user name, and password **for connect** to z/OS using FTP."

File: docs/user-guide/cli-ftpplugin.md [hunk 3]
```diff
@@ -133,5 +114,5 @@ For example, you can use one of the following methods to download a data set:
   ```
 - Download a data set without using a default profile:
   ```
-  zowe zftp download data-set USERHLQ.DATASET.NAME --host <hostname> --port 21 --user <User_ID> --password <password> --secure-ftp false
```
Comment (anaxceron) [inline docs/user-guide/cli-ftpplugin.md:136]: Line 107: plug-in is hyphenated

**Review by anaxceron — APPROVED** [review]
Lookin' good, @zFernand0. Just a couple of minor notes.

**Comment by zFernand0** [comment]
Hey @awharn and @t1m0thyj,
Should we include any other plug-in updates in this PR? 😋

**Comment by awharn** [comment]
> Hey @awharn and @t1m0thyj,
> Should we include any other plug-in updates in this PR? 😋 

DB2 has its own PR.

**Review by anaxceron — APPROVED** [review]
awesome!

**Pull request merged**